package.json

```diff
@@ -2,11 +2,15 @@
   "name": "react-responsive-picture",
   "version": "1.0.1",
   "description": "A future-proof responsive image component that supports latest Picture specification",
-  "main": "./dist/index.js",
+  "main": "./lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
```

Comment:
Yes I believe you're right @bobbyrenwick, I was just using other libraries as reference but it seems that `jsnext:main` can be safely dropped. As a future reference I think [this thread](https://www.reddit.com/r/javascript/comments/5jwg9c/confused_about_fields_module_and_jsnextmain_in/) also helps clarifying why it is dropped in favour of `module`.